Shoko.Server/API/v3/Models/Shoko/WebUI.cs

```diff
@@ -217,6 +217,14 @@ public WebUISeriesFileSummary(SVR_AnimeSeries series, HashSet<EpisodeType> episo
                     if (file?.Media == null || location == null)
                         return null;
 
+                    return new { file, xref, location };
+                })
+                .Where(f => f != null);
```

Comment:
at least add .ToList() after the .Where() to execute the query amd save the results in memory, so we don't have to run it twice. 😔

Reply:
```suggestion
                .Where(tuple => tuple.file != null);
```